### DISCUSSION TITLE
Can I emit a comment when writing yaml?

### DISCUSSION THREAD

**Discussion by gdhoupis** [biojppm/rapidyaml#437] [Q&A]
Can I emit a comment when writing yaml?  I am trying to generate a yaml output of a configuration and I would like to embed some comment lines in the output to help annotate the meaning for a human reader.  Is there a way to embed a comment like node in the output tree NodeRef?  One hacky workaround I am contemplating is writing a dummy key/val with key="MAKECOMMENT" and then write the yaml to a buffer and replace all "s/MAKECOMMENT/# /g".

**Comment by biojppm** [comment]
Sorry, there is nothing in the library to do that.
